main.go

```diff
@@ -40,6 +40,7 @@ import (
 	kcollectors "k8s.io/kube-state-metrics/pkg/collectors"
 	"k8s.io/kube-state-metrics/pkg/options"
 	"k8s.io/kube-state-metrics/pkg/version"
+	"k8s.io/kube-state-metrics/pkg/whiteblacklist"
```

Comment:
Can we already call this allow/deny list as planned for 2.0 and just keep the flags called as they are called today?

Reply:
@brancz I would prefer to do this in one go to avoid confusion. Is that also ok?

Reply:
also fine by me